### ISSUE TITLE
Language Switcher should be reachable via keyboard navigation

### ISSUE THREAD

**Issue by lomky** [cagov/ui-claim-tracker#389]
### Description
All links on the site should be reachable viaa the keyboard. Right now when the site is tabbed through the Language Switcher is not reachable.

### Acceptance Criteria

- [ ] Language Switcher is reachable via tab navigation
